### ISSUE TITLE
[website module request] Mangallama

### ISSUE THREAD

**Issue by azraei97** [dazedcat19/FMD2#279]
**Website Information**
* **Name:** Mangallama
* **Root URL:** https://mangallama.com
* **Directory list URL:** https://mangallama.com/browse.php

* **Example manga info URL:** https://mangallama.com/manga.php?manga=My-Girlfriend-Is-A-Villain
* **Example chapter URL:** https://mangallama.com/readmanga.php?manga=747&ch=126813&pg=all

**Languages**
List of languages supported by the website

**Website relationship**
If applicable describe the relation with any other website (eg : "alternative domain, copy of ...")

**Additional details**
Add any other context details that you may have found (like template to reuse)

**Comment by Slasar41** [comment]
Dead site.

**Comment by azraei97** [comment]
Lol. Request since 2021 yet reply now? That group already gone since 3 years ago.

**Comment by Slasar41** [comment]
Yeah I just have a time to start checking older issue one by one.
